### DISCUSSION TITLE
ResultOfVerification Field

### DISCUSSION THREAD

**Discussion by DeanofHouseHanson** [OP-TED/eForms-SDK#948] [Q&A] [answered]
Hi all,

I'm attempting to post a PIN notice (notice type 4) and running into a schema validation issue related to the ResultOfVerification field.

{"code":"notice.CVS_VALIDATION_FAILED","message":"CVS service failed to validate the notice. The notice is an invalid XML against its XSD. See below the full error:\n[{\"line\":95,\"column\":25,\"message\":\"cvc-complex-type.2.4.a: Invalid content was found starting with element '{\\\"urn:oasis:names:specification:ubl:schema:xsd:CommonAggregateComponents-2\\\":Attachment}'. One of '{\\\"urn:oasis:names:specification:ubl:schema:xsd:CommonAggregateComponents-2\\\":ResultOfVerification}' is expected.\"}]","showDetails":false}

Below is a snippet of what the validation error is referring to:
![image](https://github.com/OP-TED/eForms-SDK/assets/122117628/272f27f2-0f42-498e-9375-b9b26e60d466)

There only seems to be one reference to this field in the 1.11 SDK in the xsd schema file:
![image](https://github.com/OP-TED/eForms-SDK/assets/122117628/368eb4af-e7fd-4cd2-b305-9e01f1f93372)

Meaning that it isn't referenced in any of the notice type files or in the fields.json file.

Has this been brought up before? / Is there a reason that the field it is expecting is not in the notice templates?

Thanks in advance!

**Comment by bertrand-lorentz** [comment, marked as answer]
The error in your XML is that `cac:Attachment` must be before `cac:IssuerParty`
This can been seen in the schema at:
https://github.com/OP-TED/eForms-SDK/blob/develop/schemas/common/UBL-CommonAggregateComponents-2.3.xsd#L2159

The suggestion in the second part of the message is a bit misleading: the schema validation sees the `cac:IssuerParty` followed by `cac:Attachment` and indicates that the only element allowed after `cac:IssuerParty` is `cac:ResultOfVerification`.
But `cac:ResultOfVerification` is not used in eForms.
This message comes from the standard XML library, so we don't have any control over it.

The information indicated in the "xsdSequenceOrder" property in fields.json can help you determine the right sequence for the various XML elements.
See: https://docs.ted.europa.eu/eforms/latest/guide/xml-generation.html#_getting_the_xml_element_order_right
